test/evaluate.js

```diff
@@ -108,6 +108,10 @@ describe('evaluate', () => {
     assert(!!point,
       `No telemetry point "evaluate" was recorded. Actual points: ${JSON.stringify(telemetry.map(p => p.name))}`)
     assert.strictEqual(point.fields.unique_tasks, '0i')
+
+    assert.strictEqual(point.fields.group_winning_min, '1')
+    assert.strictEqual(point.fields.group_winning_mean, '1')
+    assert.strictEqual(point.fields.group_winning_max, '1')
```

Comment:
This is not the best test case because the rates are the same as for an empty round. I am kind of lazy to write a better one; let me know if you think it's worth adding a test.

Reply:
fine with me